README.md

```diff
@@ -33,6 +33,14 @@ After using [CyberChef](https://gchq.github.io/CyberChef/) to experiment and fin
 - In a browser, navigate to `localhost:3000` to see usage documentation.
 
 
+### Docker
+A Docker image can be built, then run by doing the following:
+
+- `git clone https://github.com/gchq/CyberChef-server`
+- `cd CyberChef-server`
+- `docker build -t cyberchef-server .`
+- `docker run -dit --name=cyberchef-server -p 3000:3000 cyberchef-server`
```

Comment:
I think you've got a weird mix of flags on this run command, I would either choose `-it` or `-d`, but running with both doesn't make much sense. 

Seeing as this is an quick usage example, I would suggest using the flags `-it --rm`, so that the user can see the logs (and the `--rm` removes the stopped image to avoid bloat)

Reply:
Understood, I will take a look at it.

Reply:
Okay, I went ahead and changed this verbiage in the PR.